UndertaleModCli/Program.UMTLibInherited.cs

```diff
@@ -521,13 +521,13 @@ public string SimpleTextInput(string title, string label, string defaultValue, b
     }
 
     /// <inheritdoc/>
-    public async Task ClickableSearchOutput(string title, string query, int resultsCount, IOrderedEnumerable<KeyValuePair<string, List<string>>> resultsDict, bool editorDecompile, IOrderedEnumerable<string> failedList = null)
+    public async Task ClickableSearchOutput(string title, string query, int resultsCount, IOrderedEnumerable<KeyValuePair<string, List<(int, string)>>> resultsDict, bool editorDecompile, IOrderedEnumerable<string> failedList = null)
```

Comment:
Seeing a tuple here feels very weird to me right now, but I currently can't think of something nicer. 
In the meantime, can you at least use names for them? `List<(int lineNumber, string codeSnippet)>` or something